pkg/controller/template/render.go

```diff
@@ -370,6 +379,13 @@ func cloudConfigFlag(cfg RenderConfig) interface{} {
 		}
 	}
 
+	external, err := cloudprovider.IsCloudProviderExternal(cfg.Infra.Status.Platform, cfg.FeatureGate) //nolint
+	if err != nil {
+		glog.Error(err)
```

Comment:
same as above - does only logging this error make sense?

Reply:
https://github.com/openshift/machine-config-operator/pull/2386#discussion_r587561388